pjlib/include/pj/compat/socket.h

```diff
@@ -1,3 +1,8 @@
+/**
```

Comment:
One star again - I won't comment on this from here on, as I'm not paid by the comment 😛